app/models/ability.rb

```diff
@@ -27,6 +27,7 @@ def initialize(user)
 
     if user.admin?
       can :manage, :all
+      can :show, :all_schools
```

Comment:
I think this is inferred by the `:manage, :all` above

Reply:
Doh! Sorted...